src/xprof_elixir_syntax.erl

```diff
@@ -0,0 +1,273 @@
+%%%
+%%% @doc Module to parse and format expressions in Elixir syntax
+%%%
+-module(xprof_elixir_syntax).
+
+-behaviour(xprof_language).
+
+-export([parse_query/1,
+
```

Comment:
I just wanted to emphasize division between callbacks for query parsing and callbacks for auto-compete. But probably there are other sublte signs to divide them anyway.